Userland/Libraries/LibCore/System.cpp

```diff
@@ -680,6 +680,14 @@ ErrorOr<bool> isatty(int fd)
     return rc == 1;
 }
 
+ErrorOr<String> ttyname(int fd)
```

Comment:
This could be a StringView because `ttyname` returns a pointer to a static buffer